ui/app/components/job-status/allocation-status-block.hbs

```diff
@@ -3,6 +3,13 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
+{{!--
+  Exists as a "middleman" between AllocationStatusRow and IndividualAllocation
+  only when showSummaries in AllocationStatusRow is true (i.e. when the math of how
+  many allocations to show in each block is done x minWidth of each alloc exceeds
+  available space)
+--}}
+
```

Comment:
Side-effect: I should've left this comment during the original job-panel redesign, but figured I'd add it now.